_rules/6cbcbe.md

```diff
@@ -0,0 +1,280 @@
+---
+id: b20e66
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names resolve to the same resource or equivalent resources.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names](#accessible-name) that do not only consist of [whitespace](#whitespace).
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name](#accessible-name).
+
+### Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource](#same-resource) or to [equivalent resources](#equivalent-purpose).
+Resolving the links includes potential redirects, if the redirects happen instantly.
```

Comment:
This seems like it should be a note. Either add a new line between the two paragraphs, or put them on the same line.

Reply:
Removed the break between the two paragraphs. Is that enough to satisfy your feedback? 

If we move it into a note, it will be informative, and it shouldn't be. It's part of what makes the expectation.

Reply:
In that case yeah, remove the line break. Either have something on the same line, or have an empty line between.